### DISCUSSION TITLE
It worked and then stopped

### DISCUSSION THREAD

**Discussion by thekrabajes96** [aclap-dev/video-downloadhelper#491] [License & Purchase requests]
### What's your issue?

I have mozzilla license. I have every thing set but it stopped working please help

### The 10 first digits (only 10) of your license number or email address

fd270e8c69

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
I checked your license, it's still valid. What is not working?

**Comment by thekrabajes96** [comment]
I am not sure is there a way you can check?

**Reply by jcfrog** [reply]
Sorry I don't understand.
I confirm your license has been validated 4 times and is still valid. But we need to know more about your issue.

**Comment by thekrabajes96** [comment]
I am not able to download the videos of my choice I choose the download
options and then it says there is an error. One video only worked and then
started giving me error. This is in Mozzila.
I also have a chrome license and it does not work for anything!
Please help!

**Reply by jcfrog** [reply]
please share your versions : https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F
And a URL of a public video with this issue should help.

**Comment by thekrabajes96** [comment]
version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): fd270e8c698e4f85
platform: aarch64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:125.0) Gecko/20100101
Firefox/125.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/Applications/
net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "mac",
    "arch": "arm64",
    "node": "18"
  },
  "home": "/Users/thekrabajes",
  "converterBinary": "/Applications/
net.downloadhelper.coapp.app/Contents/MacOS/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "n6.1-11-gc30f360955"
}



I am not trying to download a public video I am trying to download videos
of a course I am taking right now.

**Comment by thekrabajes96** [comment]
It gives me this code

***@***.***://b96b75c3-6964-4f73-80e8-931ebfafadb2/background/main.js:1:13157
***@***.***://b96b75c3-6964-4f73-80e8-931ebfafadb2/background/main.js:1:42626

**Reply by paulrouget** [reply]
There's probably more to it. You're missing part of the error.

**Comment by thekrabajes96** [comment]
Can you please help me fix it?

**Comment by thekrabajes96** [comment]
Is there a way you can view my screen and fix it

**Reply by paulrouget** [reply]
Install Version Beta: https://www.downloadhelper.net/beta

Then try again.

It will probably fail, so click on the Copy button, and paste the result here.

**Comment by OranCN** [comment]
다운로드가 정상 진행 되지 않는다.
1시간 미만 영상만 정상 다운되고 1시간이상 동영상은 다운로드 오류가 발생한다.

After registering for premium code
The download is not proceeding normally.
